README.md

```diff
@@ -27,7 +27,19 @@ Instructions coming soon!
 
 ## To develop gRPC for ASP.NET Core
 
-TODO: add development instructions
+Install [.NET Core SDK 3 preview 2](https://dotnet.microsoft.com/download/dotnet-core/3.0)
+or [VS 2019 preview](https://visualstudio.microsoft.com/vs/preview/).
```

Comment:
I'm pretty sure VS2019 doesn't ship with .NET Core 3 preview 2.

Reply:
Ok, so there's currently no way to build with VS? (I haven't tried yet).

Reply:
VS2017 or VS2019 should work at the moment once you have downloaded preview 2.

I'm not sure about VS Code, I don't use it myself. @davidfowl are you using VS Code?

Reply:
I'm using VS 2019